### DISCUSSION TITLE
Why "ConstructorExtractor" from symfony/property-info is not registered by framework-bundle?

### DISCUSSION THREAD

**Discussion by andrew-demb** [symfony/symfony#58551] [Q&A] [answered]
There's existent implementation of property type extractor - ConstructorExtract https://github.com/symfony/symfony/blob/643745fb9a7ffe9e2daab45e81eccbf2556f2898/src/Symfony/Component/PropertyInfo/Extractor/ConstructorExtractor.php#L22.
But this extractor is not registered by `framework-bundle`.

Why "ConstructorExtractor" from symfony/property-info is not registered by framework-bundle?

Refs:
- https://symfony.com/doc/current/components/property_info.html#constructorextractor
- https://github.com/symfony/symfony/issues/38416#issuecomment-841419861

**Comment by MatTheCat** [comment, marked as answer]
See https://github.com/symfony/symfony/pull/50334
